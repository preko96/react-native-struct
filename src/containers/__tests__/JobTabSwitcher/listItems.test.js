import React from 'react'
import ListItem from '../../JobsTabSwitcher/listItem'

it('renders without crashing', () => {
	expect(shallow(<ListItem/>)).toMatchSnapshot()
})