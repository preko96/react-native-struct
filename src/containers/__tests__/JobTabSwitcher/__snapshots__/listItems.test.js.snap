// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Styled(ListItem)
  icon={true}
>
  <Styled(Left)>
    <Component
      style={
        Object {
          "backgroundColor": "red",
          "borderRadius": 20,
          "height": 15,
          "width": 15,
        }
      }
    />
  </Styled(Left)>
  <Styled(Body) />
  <Styled(Right)>
    <Styled(Icon)
      name="arrow-forward"
    />
  </Styled(Right)>
</Styled(ListItem)>
`;
